doc/stolonctl.md

```diff
@@ -1,18 +1,39 @@
 ## Stolon Client (stolonctl)
 
-`stolonctl` is the stolon client which controls the stolon cluster(s)
+`stolonctl` is the stolon client which controls the stolon cluster(s) and may be executed inside a pod running a stolon component or externally. 
```

Comment:
remove this part since it's related to k8s and already explained in the k8s example: `and may be executed inside a pod running a stolon component or externally`